src/node_jsvmapi.h

```diff
@@ -18,6 +18,7 @@
 #ifndef SRC_NODE_JSVMAPI_H_
 #define SRC_NODE_JSVMAPI_H_
 
+#include "node.h"
```

Comment:
We got by without any inclusion of `node.h` before. It works because `node_jsvmapi.h` defines its own `NODE_MODULE_ABI` macro that is used for module registration. I don't think any refactoring of `node.h` is necessary.